movie/src/components/movie/Movie.css

```diff
@@ -0,0 +1,22 @@
+* {
```

Comment:
the only place to use this `*` should only be at the main `index.css` for the project. 

Could try to move this to the `.movie`